src/styles/constants/_media-queries.scss

```diff
@@ -1,10 +1,12 @@
 $size-small-max: 374px;
 $size-medium-max: 767px;
 $size-large-max: 1023px;
+$modal-fullscreen-width: 460px;
```

Comment:
Should the naming here refer to a screen size value like the other variables above? We're measuring the screen rather than the modal, right?

Reply:
Renamed to clarify its screen width and not modal width.